pkg/executor/sortexec/sort.go

```diff
@@ -772,6 +767,9 @@ func (e *SortExec) buildKeyColumns() {
 func (e *SortExec) lessRow(rowI, rowJ chunk.Row) int {
 	for i, colIdx := range e.keyColumns {
 		cmpFunc := e.keyCmpFuncs[i]
+		if cmpFunc == nil {
+			break
```

Comment:
Why there will be nil cmpFunc?

Reply:
> Why there will be nil cmpFunc?

We encounter this case in tidb integration tests.